messagebus/src/vespa/messagebus/network/rpcsend.cpp

```diff
@@ -234,7 +242,7 @@ RPCSend::doHandleReply(Reply::UP reply) {
     }
     Blob payload(0);
     if (reply->getType() != 0) {
-        payload = _net->getOwner().getProtocol(reply->getProtocol())->encode(ctx->getVersion(), *reply);
+        payload = protocol->encode(ctx->getVersion(), *reply);
```

Comment:
Will a reply without an associated protocol always have `getType() == 0`? Otherwise, it's possible to stroll in here with a `nullptr` protocol object. This isn't something introduced with this diff, so in that sense it's orthogonal to the pull request itself, but it looks a bit scary...

Reply:
The EmptyReply has empty protocol. So far the only one I found. Should I assert that getType() is 0 in that case ?

Reply:
Adding an assertion makes sense to me